src/configs/split_infer_res34.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Instead of creating new configs for each model, let's have a README section that talks about different arguments supported for the key `model_name`

Reply:
Yeah, makes sense. I will update the readme. Also, do you want me to remove these config files (I had made them for testing purposes)